2022-02-09 17:54:55,533 INFO    MainThread:3009 [wandb_setup.py:_flush():71] setting env: {}
2022-02-09 17:54:55,533 INFO    MainThread:3009 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-09 17:54:55,534 INFO    MainThread:3009 [wandb_init.py:_log_setup():371] Logging user logs to /Users/liwenyu/Documents/GitHub/RTDM/wandb/run-20220209_175455-zy8k5o5n/logs/debug.log
2022-02-09 17:54:55,534 INFO    MainThread:3009 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/liwenyu/Documents/GitHub/RTDM/wandb/run-20220209_175455-zy8k5o5n/logs/debug-internal.log
2022-02-09 17:54:55,534 INFO    MainThread:3009 [wandb_init.py:init():404] calling init triggers
2022-02-09 17:54:55,534 INFO    MainThread:3009 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-09 17:54:55,534 INFO    MainThread:3009 [wandb_init.py:init():449] starting backend
2022-02-09 17:54:55,534 INFO    MainThread:3009 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-02-09 17:54:55,552 INFO    MainThread:3009 [backend.py:ensure_launched():199] starting backend process...
2022-02-09 17:54:55,560 INFO    MainThread:3009 [backend.py:ensure_launched():204] started backend process with pid: 3025
2022-02-09 17:54:55,564 INFO    MainThread:3009 [wandb_init.py:init():458] backend started and connected
2022-02-09 17:54:55,567 INFO    MainThread:3009 [wandb_init.py:init():519] updated telemetry
2022-02-09 17:54:55,592 INFO    MainThread:3009 [wandb_init.py:init():549] communicating current version
2022-02-09 17:54:56,799 INFO    MainThread:3009 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-09 17:54:56,799 INFO    MainThread:3009 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-09 17:54:57,011 INFO    MainThread:3009 [wandb_init.py:init():592] starting run threads in backend
2022-02-09 17:55:01,690 INFO    MainThread:3009 [wandb_run.py:_console_start():1816] atexit reg
2022-02-09 17:55:01,691 INFO    MainThread:3009 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-09 17:55:01,691 INFO    MainThread:3009 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-09 17:55:01,694 INFO    MainThread:3009 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-09 17:55:01,694 INFO    MainThread:3009 [wandb_init.py:init():619] run started, returning control to user process
2022-02-09 17:55:01,712 INFO    MainThread:3009 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-02-09 17:55:01,713 INFO    MainThread:3009 [wandb_run.py:_restore():1758] restore
2022-02-09 17:55:04,722 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 712
}

2022-02-09 17:55:05,382 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 712
}

2022-02-09 17:55:06,294 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 6412
}

2022-02-09 17:55:06,397 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 6438
}

2022-02-09 17:55:06,504 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 6438
}

2022-02-09 17:55:06,612 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 712
  total_bytes: 6438
}

2022-02-09 17:55:06,715 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:06,820 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:06,933 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:07,036 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:07,138 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:07,500 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}

2022-02-09 17:55:07,975 INFO    MainThread:3009 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6438
  total_bytes: 6438
}
local_info {
}

2022-02-09 17:55:09,413 INFO    MainThread:3009 [wandb_run.py:_append_files():2184] logging synced files
